src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -293,14 +293,18 @@ def _write_tsv(self, output: IO[str]) -> None:
 
     def _create_bdbag_archive(self) -> str:
         with TemporaryDirectory() as bag_path:
-            bdbag_api.make_bag(bag_path)
-            with open(os.path.join(bag_path, 'data', 'samples.tsv'), 'w') as samples_tsv:
+            manifest_folder = os.path.join(bag_path, 'manifest')
```

Comment:
Rename `bag_path` to `temp_path` and `manifest_folder` to `bag_path`. I want to avoid the overloading the term "manifest" with yet another term.